src/pages/documentation/migrate-to-android.md

```diff
@@ -0,0 +1,89 @@
+| Campaign        | [![Maven Central](https://img.shields.io/maven-central/v/com.adobe.marketing.mobile/campaign.svg?logo=android&logoColor=white&label=campaign)](https://mvnrepository.com/artifact/com.adobe.marketing.mobile/campaign)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
+| CampaignClassic | [![Maven Central](https://img.shields.io/maven-central/v/com.adobe.marketing.mobile/campaignclassic.svg?logo=android&logoColor=white&label=campaignclassic)](https://mvnrepository.com/artifact/com.adobe.marketing.mobile/campaignclassic)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
+
+### Manual library import
+
+If you are manually importing SDK libraries, ensure you identify all currently used Android 1.x libraries and switch them over to Android 2.0 libraries. The list of current Android 2.0 libraries can be found in the [current SDK versions document](sdk-versions.md#android) (in the Android section).
+
+### Gradle
+
+If you are using Gradle to manage your Adobe Experience Platform Mobile SDK dependencies, the following example shows you how to switch Android 1.x libraries to Android 2.0 libraries in your `build.gradle` file.
+
+```java
+dependencies {
+    //replace sdk-core with Core/Lifecycle/Identity/Signal 2.0
+    //implementation 'com.adobe.marketing.mobile:sdk-core:1.+'
+    implementation 'com.adobe.marketing.mobile:core:2.+'
+    implementation 'com.adobe.marketing.mobile:identity:2.+'
+    implementation 'com.adobe.marketing.mobile:lifecycle:2.+'
+    implementation 'com.adobe.marketing.mobile:signal:2.+'
+
+    // replace UserProfile 1.+ with UserProfile 2.0
+    //implementation 'com.adobe.marketing.mobile:userprofile:1.+'
+    implementation 'com.adobe.marketing.mobile:userprofile:2.+'
+
+}
+```
+
+<InlineAlert variant="warning" slots="text"/>
```

Comment:
can we move this one before the code sample for visibility?

Reply:
I intend on showing the alert here since the following table shows `sdk-core` should be replaced with `core/identity/lifecycle/signal`.